practice_flow_control_and_methods/day_of_the_week_checker.rb

```diff
@@ -0,0 +1,23 @@
+# Ask for a day.
+
+# If "Saturday" or "Sunday" → “It’s the weekend!”
+# Otherwise → “It’s a weekday!”
+
+puts "'what day is it today?' (monday, tuesday, wednesday, thursday, friday, saturday, sunday)"
+given_day = gets.chomp.downcase.to_s
+
+weekday_array = ["monday", "tuesday", "wednesday", "thursday", "friday"]
+weekend_array = ["saturday", "sunday"]
+
+if weekend_array.include?(given_day)
+  puts "it's the weekend!"
+elsif weekday_array.include?(given_day)
```

Comment:
L12 checks if the given_day is a weekend. since we are not on L14 we know that the given day has to be a weekday, so this check is redundant. that way you can flatten L16-L20 (the nested `if`)